src/main/resources/hibernate.cfg.xml

```diff
@@ -0,0 +1,19 @@
+<hibernate-configuration>
+    <session-factory>
+        <property name="connection.driver_class">com.mysql.cj.jdbc.Driver</property>
+        <property name="connection.url">jdbc:mysql://localhost:3306/cinema?serverTimezone=UTC</property>
+        <property name="connection.username">root</property>
+        <property name="connection.password">maks</property>
+        <property name="dialect">org.hibernate.dialect.MySQL8Dialect</property>
+        <property name="show_sql">true</property>
+        <property name="hbm2ddl.auto">create-drop</property>
+
+        <mapping class="mate.academy.model.Movie"/>
+        <mapping class="mate.academy.model.MovieSession"/>
+        <mapping class="mate.academy.model.CinemaHall"/>
+        <mapping class="mate.academy.model.User"/>
+        <mapping class="mate.academy.model.Ticket"/>
+        <mapping class="mate.academy.model.ShoppingCart"/>
+        <mapping class="mate.academy.model.Order"/>
+    </session-factory>
+</hibernate-configuration>
```

Comment:
add empty line in the end, Kostiya commented the same issue in my work today :)

Reply:
![image](https://user-images.githubusercontent.com/61313877/218040027-a5ed3f1a-1023-43f8-81d6-2efe10d25e36.png)
already have, idk what git want from me

Reply:
Here everything is okay